src/slices/EmailSignup.js

```diff
@@ -1,32 +1,26 @@
 import * as React from 'react'
 import { graphql } from 'gatsby'
-import { RichText } from 'prismic-reactjs'
-
-import { GatsbyLink } from '../components/GatsbyLink'
+import { PrismicRichText } from '@prismicio/react'
+import { asText } from '@prismicio/helpers'
 
 export const EmailSignup = ({ slice }) => (
   <section className="email-signup">
     <div className="description">
-      <RichText render={slice.primary.section_title.richText || []} />
-      <RichText
-        render={slice.primary.description.richText || []}
-        serializeHyperlink={GatsbyLink}
-      />
+      <PrismicRichText field={slice.primary.section_title.richText} />
+      <PrismicRichText field={slice.primary.description.richText} />
     </div>
     <div className="form">
-      <RichText render={slice.primary.input_label.richText || []} />
+      <PrismicRichText field={slice.primary.input_label.richText} />
       <input
         className="email-input"
         type="text"
         name="FirstName"
-        placeholder={RichText.asText(
-          slice.primary.input_placeholder.richText || [],
-        )}
+        placeholder={asText(slice.primary.input_placeholder.richText || [])}
```

Comment:
why did you leave the `||` operator here and not in the rich texts?

Reply:
`asText()` doesn't allow `null` or `undefined`, but `<PrismicText>` and `<PrismicRichText>` does.

I asked @libhr about updating `asText()` to work like `<PrismicText>` and she agrees. I'll update `@prismicio/helpers` with this change. Once it's published, we can change this line to this:

```javascript
placeholder={asText(slice.primary.input_placeholder.richText)}
```

I'll make a separate PR for this.